OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
It wasn't me who edited that comment. And I think someone who actually starts out with dismissing a polite comment on an offensive name at start because of carelessness is pretty much asking for it.